convoys/regression.py

```diff
@@ -278,20 +278,16 @@ def callback(LL, value_history=[]):
             'beta': data[6+n_features:6+2*n_features].T,
         } for k, data in result.items()}
 
-    def cdf(self, x, t, ci=None):
+    def cdf_posteriori(self, x, t, ci=None):
```

Comment:
Is this a name that's used elsewhere? Was thinking it's not really a posteriori distribution, but samples from the posterior distribution – maybe pick a name to make that clear?

Reply:
Yeah I think thats fair, it is the samples themselves.

Reply:
We can rename it at some other point. I think the "cdf" naming is really confusing already – should probably just be called `predict` or something to be consistent with scikit-learn.